features/storage-manager.yml

```
@@ -1,11 +1,11 @@
-draft_date: 2024-09-09
-name: Storage Standard
-description: TODO
+name: Storage manager
+description: "The `navigator.storage` API provides information about the availability and persistence of data stored by the browser."
```

```suggestion
description: "The `navigator.storage` API provides information about the availability and persistence of the data that a site stores on the device, by using APIs such as the Cache API or the IndexedDB API."
```

I feel like this is a bit clearer this way. I'm not certain about which APIs to mention though, as this has been a moving target. Eventually, _all_ client-side stored data will be managed through by the Storage Manager, but I don't think this is the case currently.